code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -1164,6 +1164,24 @@ All effects don't start immediately, but rather get worse over time; the rate is
 		. = 1
 	return ..() || .
 
+/datum/reagent/consumable/ethanol/bureau
```

Comment:
I would like this to be in the Monkestation module files instead.
'monkestation/code/modules/reagents/chemistry/reagents/alcohol_reagents.dm'